jetty-websocket/websocket-core-server/src/main/java/org/eclipse/jetty/websocket/core/server/internal/UpgradeHttpServletRequest.java

```diff
@@ -276,56 +288,50 @@ public HttpSession getSession()
         return session;
     }
 
-    public Request getBaseRequest()
-    {
-        return baseRequest;
-    }
-
     @Override
     public String getRequestedSessionId()
     {
-        throw new UnsupportedOperationException(UNSUPPORTED_WITH_WEBSOCKET_UPGRADE);
+        throw new UnsupportedOperationException(UNSUPPORTED_AFTER_WEBSOCKET_UPGRADE);
```

Comment:
All these methods could use the request object while not null and only throw once it is null.